app/assets/javascripts/views/project_view.js

```diff
@@ -1,9 +1,10 @@
+import AttachmentOptions from 'models/attachmentOptions'
+import StoryAttachment from 'components/story/StoryAttachment';
 var StoryView = require('./story_view');
 var IterationView = require('./iteration_view');
 var ColumnView = require('./column_view');
 var ColumnVisibilityButtonView = require('./column_visibility_button_view');
 var HistoryView = require('./history_view');
-
```

Comment:
I think this is a good space, can we keep it?

Reply:
done, and space back.